O365Troubleshooters.psm1

```diff
@@ -1581,6 +1581,7 @@ Function Start-O365TroubleshootersMenu {
     11 Tools: Decode SafeLinks URL
     12 Tools: Export Quarantine Messages
     13 Tools: Transform IMCEAEX (old LegacyExchangeDN) to X500 address
+    14 Groups: DL to O365 Groups Upgrade Checker
```

Comment:
please add your diagnostic at position 5 in the main menu and increment the others after that

Reply:
done!